ecommerce/views.py

```diff
@@ -202,3 +205,33 @@ def get(self, request, username, *args, **kwargs):  # pylint: disable=unused-arg
         )
 
         return Response(serialize_user_bootcamp_runs(user=user))
+
+
+class CheckoutDataViewSet(ReadOnlyModelViewSet):
```

Comment:
This view should just return a single item now, shouldn't it? It's returning the correct data for me, but it's in a single item list and I don't think it needs to be

Reply:
Sure, just made the change